src/test/javascript/spec/component/quiz-exercise/quiz-exercise-visual-editor.component.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { ComponentFixture, TestBed, fakeAsync } from '@angular/core/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { CourseManagementService } from 'app/course/manage/course-management.service';
+import { of } from 'rxjs';
+import { MockRouter } from '../../helpers/mocks/mock-router';
+import { MockTranslateService } from '../../helpers/mocks/service/mock-translate.service';
+import { MockComponent, MockDirective, MockModule, MockPipe, MockProvider } from 'ng-mocks';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { ArtemisNavigationUtilService } from 'app/utils/navigation.utils';
+import { MultipleChoiceVisualQuestionComponent } from 'app/exercises/quiz/shared/questions/multiple-choice-question/multiple-choice-visual-question.component';
+import { MultipleChoiceQuestion } from 'app/entities/quiz/multiple-choice-question.model';
+import { AnswerOption } from 'app/entities/quiz/answer-option.model';
+import { FaIconComponent } from '@fortawesome/angular-fontawesome';
+import { NgModel } from '@angular/forms';
+import { NgbTooltipModule } from '@ng-bootstrap/ng-bootstrap';
+
+describe('QuizVisualEditorComponent', () => {
+    let fixture: ComponentFixture<MultipleChoiceVisualQuestionComponent>;
+    let comp: MultipleChoiceVisualQuestionComponent;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [MockModule(NgbTooltipModule)],
+            declarations: [MultipleChoiceVisualQuestionComponent, MockPipe(ArtemisTranslatePipe), MockComponent(FaIconComponent), MockDirective(NgModel)],
+            providers: [
+                MockProvider(ArtemisNavigationUtilService),
+                MockProvider(CourseManagementService),
+                { provide: TranslateService, useClass: MockTranslateService },
+                { provide: Router, useClass: MockRouter },
+                {
+                    provide: ActivatedRoute,
+                    useValue: { queryParams: of({}) },
+                },
+            ],
+            schemas: [],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(MultipleChoiceVisualQuestionComponent);
+                comp = fixture.componentInstance;
+
+                comp.question = new MultipleChoiceQuestion();
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', fakeAsync(() => {
+        fixture.detectChanges();
+        expect(comp).not.toBeNull();
+    }));
+
+    it('parse the given question properly to markdown', fakeAsync(() => {
+        fixture.detectChanges();
+        expect(comp).not.toBeNull();
```

Comment:
This is not needed, same in the other test. The beforeEach above will already fail if comp is really null